app/views/upload_your_files.scala.html

```diff
@@ -41,7 +41,7 @@
     }
 }
 
-@mainTemplate(title = Title(s"fileUploadPage.heading.$positionModifier")) {
+@mainTemplate(title = Title(s"fileUploadPage.heading.$positionModifier"), backLinkUrl = Some(routes.HowManyFilesUploadController.onPageLoad.url)) {
```

Comment:
A usability issue rather than a technical one. If a user is uploading their 3rd file and hits the back button they will be taken back to the 'How Many Files to Upload' page and have to re-upload the two files they just uploaded. I'm wondering if this will be obvious to them (probably not) should there be some sort of warning/explanation?

Reply:
Thanks, Tim. I'll report your point which I think need further discussions.

Reply:
OK. Jonathan will raise this concern at the next Refinement session.